local-cluster/src/local_cluster.rs

```diff
@@ -250,6 +252,20 @@ impl LocalCluster {
             }
         };
 
+        let core_programs = core_bpf_programs(&Rent::default(), |_| true);
+        let stake_idx = core_programs
+            .iter()
+            .position(|(key, _)| *key == solana_stake_program::id())
+            .unwrap();
+        let stake_program = core_programs[stake_idx].1.clone();
+        let (stake_programdata_address, stake_programdata) = core_programs[stake_idx + 1].clone();
+        config
+            .additional_accounts
+            .push((solana_stake_program::id(), stake_program));
+        config
+            .additional_accounts
+            .push((stake_programdata_address, stake_programdata));
+
```

Comment:
im concerned this is a hack, i dont know what mechanism a real cluster genesis uses to make sure it gets the core bpf programs, and if it is viable for `LocalCluster` to use it as well. i guess my view is "if we have some kind of test coverage of the multinode setup or similar, that ensures fresh clusters actuallu work, what we do in `LocalCluster` doesnt really matter. if `LocalCluster` is the only test surface we have over setting up clusters, probably we should use the formal mechanism that real clusters use to get core bpf programs"

after i pass ci ill probably change this to load in all the core bpf programs just for sake of completeness. i assume because it goes through `GenesisConfig` like bank tests, we are implicitly `FeatureSet::all_enabled()`